hawkular-inventory-parent/hawkular-inventory-service/src/test/java/org/hawkular/inventory/service/InventoryServiceIspnTest.java

```diff
@@ -323,5 +332,11 @@ public void shouldGetLargeExport() throws IOException {
                 .hasSize(maxTypes + 3)
                 .extracting(ResourceType::getId)
                 .contains("EAP", "RT0", "RT100", "RT199");
+
+        InventoryHealth health = service.getHealthStatus();
+        assertThat(health.getInventoryStats()).isNotNull();
+        assertThat(health.getInventoryStats().getNumberOfResources() > 10000);
```

Comment:
Here, no actual assertion is made... that would be `assertThat(health.getInventoryStats().getNumberOfResources()).isGreaterThan(10000)` , and similar below.

PS: I introduced assertj during the POC as I think it's an awesome lib, especially when dealing with collections, but I hope it doesn't cause frustration... If so let's talk about it

Reply:
Ok, I will re-arrange this to the assertj syntax.